src/nninf/qual/NonNull.java

```diff
@@ -1,38 +1,34 @@
 package nninf.qual;
 
-import org.checkerframework.framework.qual.DefaultFor;
-import org.checkerframework.framework.qual.LiteralKind;
-import org.checkerframework.framework.qual.QualifierForLiterals;
-import org.checkerframework.framework.qual.SubtypeOf;
-import org.checkerframework.framework.qual.TypeKind;
-import org.checkerframework.framework.type.AnnotatedTypeMirror.AnnotatedPrimitiveType;
+import org.checkerframework.framework.qual.*;
```

Comment:
Please do not use a wild card with a Java import statement.

Reply:
Thanks for pointing that out!